recipe/meta.yaml

```diff
@@ -2,34 +2,57 @@
 
 {% set version = "29.3" %}
 
+# Version of GCC sources used to build the embedded JIT compiler
+{% set gcc_version = "14.2.0" %}
+
 package:
   name: emacs
   version: {{ version }}
 
 source:
-  fn: emacs-{{ version }}.tar.xz
-  url: http://ftp.gnu.org/gnu/emacs/emacs-{{ version }}.tar.xz
-  sha256: c34c05d3ace666ed9c7f7a0faf070fea3217ff1910d004499bd5453233d742a0
-  patches:
-    - 0001-disable-sanity-check.patch  # [osx and build_platform != target_platform]
-    - 0002-apple-silicon-resign-binary.patch  # [osx]
-    - 0003-macos-cross-compile-configure.patch  # [osx and build_platform != target_platform]
-    - 0004-macos-cross-compile-makefile.patch  # [osx and build_platform != target_platform]
-    - 0005-macos-cross-compile-nextstep-makefile.patch  # [osx and build_platform != target_platform]
-    - 0006-macos-cross-compile-post-install-pdump-path.patch  # [osx and build_platform != target_platform]
-    - 0007-macos-cross-compile-lisp-makefile.patch  # [osx and build_platform != target_platform]
+  - fn: emacs-{{ version }}.tar.xz
+    url: http://ftp.gnu.org/gnu/emacs/emacs-{{ version }}.tar.xz
+    sha256: c34c05d3ace666ed9c7f7a0faf070fea3217ff1910d004499bd5453233d742a0
+    patches:
+      - 0001-disable-sanity-check.patch  # [osx and build_platform != target_platform]
+      - 0002-apple-silicon-resign-binary.patch  # [osx]
+      - 0003-macos-cross-compile-configure.patch  # [osx and build_platform != target_platform]
+      - 0004-macos-cross-compile-makefile.patch  # [osx and build_platform != target_platform]
+      - 0005-macos-cross-compile-nextstep-makefile.patch  # [osx and build_platform != target_platform]
+      - 0006-macos-cross-compile-post-install-pdump-path.patch  # [osx and build_platform != target_platform]
+      - 0007-macos-cross-compile-lisp-makefile.patch  # [osx and build_platform != target_platform]
+  - fn: gcc-{{ gcc_version }}.tar.xz
+    url: https://ftp.gnu.org/gnu/gcc/gcc-{{ gcc_version }}/gcc-{{ gcc_version }}.tar.xz
+    sha256: a7b39bc69cbf9e25826c5a60ab26477001f7c08d85cec04bc0e29cabed6f3cc9
+    folder: gcc
 
 build:
-  number: 1
+  number: 3
   skip: true  # [win]
-  detect_binary_files_with_prefix: true
+
+  # Disable prefix autodetection on Linux to avoid patching of GCC
+  # binaries, which renders them inoperable. Adjust the sysroot and
+  # library paths using the specs file instead.
+  #
+  # See also the gcc_impl_linux-64 recipe for reference.
```

Comment:
Is there a way to just exclude those files from the automatic detection? Based on my past experience with this package the set of hardcoded paths below is very likely to become outdated whenever emacs does a major release.

Reply:
Replaced with ignore_prefix_files. I cannot make the osx build work yet so I moved it to a.separate branch for later.